config/config.go

```diff
@@ -54,13 +54,57 @@ func (d *Database) Verify() error {
 	return nil
 }
 
+type QueryParser struct {
+	Engine string `yaml:"engine,omitempty"`
+}
+
+func (q *QueryParser) Verify() error {
+	if q.Engine == "" {
+		q.Engine = "promql"
```

Comment:
since you are using this string multiple times, it will be useful to have it in a constant, and certainly a public one.